src/core/qfield.h

```diff
@@ -0,0 +1,21 @@
+#ifndef QFIELD_H
+#define QFIELD_H
+
+template <typename T>
+class asKeyValueRange
```

Comment:
Can we put this into a namespace (like `qgis.h`)?

Reply:
Oh, and a short "How to use" as docstring would be super cool

Reply:
Done